packages/mds-jurisdiction-service/migrations/1582294819607-CreateJurisdictionsTable.ts

```diff
@@ -0,0 +1,41 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class CreateJurisdictionsTable1582294819607 implements MigrationInterface {
```

Comment:
I'm curious why the class has this string of digits in it.

Reply:
The classname is generated by TypeORM. The digits represent a timestamp.

Reply:
Epoch MS... is this just to keep migrations unique? Is this file autogenerated?

Reply:
Whoops, late comment